### ISSUE TITLE
list possible actions

### ISSUE THREAD

**Issue by delanym** [syscolabs/kasaya#20]
Instead of creating masses of documentation on the grammar, it would be better to get a list of pressable keys and clickable elements and/or provide tab auto-completion
```
list keys
list clickable
```

**Comment by randilu** [comment]
@delanym
An improved version of current `help` command will cater this,
1. Kasaya currently supports all the basic typing/navigation/control keys.
Control keys:
ex:- `backspace`,`tab`,`enter`,`shift`,`control`,`alt`,`pause`,`escape`,`delete`,`end`,`home`,`insert`
Navigation keys:
`arrowleft`,`arrowright`,`arrowup`,`arrowdown`

2. `list clickable` would not be practical here. Please refer README.md for more info

**Comment by DrumApHead** [comment]
Hello All,

I would like to know if it's possible to have a combination control key + key

For instance Alt+F1

**Comment by randilu** [comment]
@DrumApHead 
Yes, you can have key combinations as well. Mentioned above are the currently supported control keys. Also all the alphanumeric character keys are supported.

ex:- `press "control+a"` 

But currently keys such as `F1` are not supported
